plugins/woocommerce-blocks/assets/js/blocks/checkout/test/block.js

```diff
@@ -104,6 +104,10 @@ describe( 'Testing cart', () => {
 
 	it( 'Renders checkout if there are items in the cart', async () => {
 		render( <CheckoutBlock /> );
+
+		// TODO: Fix a recent deprecation of showSpinner prop of Button called in this component.
+		expect( console ).toHaveWarned();
```

Comment:
`@wordpress/jest-console` automatically causes all tests with warnings to fail. I consider this TODO with assertion to be the most pragmatic way to fix this for now. We could follow up in a cooldown with a clean up task to have these components render the spinner instead.